api/locations/location.mock.ts

```diff
@@ -0,0 +1,42 @@
+/* tslint:disable:max-line-length */
+import MockAdapter from 'axios-mock-adapter';
+import {HttpMockAdapter} from '../common/httpMockAdapter';
+import {ALL_COURT_LOCATIONS} from './data/location.mock.data';
+import {toEpimsLocation} from "./models/location.model";
```

Comment:
just needs a bit of consistency, double-quotes really should've been picked up by your IDE 😅

Reply:
Merged from https://github.com/hmcts/rpx-xui-webapp/tree/feature/EUI-4690-location-api-to-master - not part of the changes in this PR.